### DISCUSSION TITLE
Jet Mode

### DISCUSSION THREAD

**Discussion by WojtekWro** [nVuln/homebridge-lg-thinq#219] [Q&A]
Hi @nVuln 
Is there a chance to disable/remove "Jet Mode" switch from homebridge? 
I do not use it at all. This is not an issue, however my node-red send me warnings and error. 
I have two ACs. 

WARNING: Duplicate device name Jet Mode - Switch
ERROR: Parsing failed, duplicate uniqueID. Jet Mode - Switch

**Comment by nVuln** [comment]
you can fork this repo and remove whatever you want
I got a lot request to remove abc...xyz feature, I can't serve all of this

**Reply by WojtekWro** [reply]
No problem. I can live with that ;) 
I appreciate your work and suggestion, but I'm not able to do that:( 

thank again
